### ISSUE TITLE
i work nice but too slow

### ISSUE THREAD

**Issue by weisuogou** [MrJoy/UnitySVG#9]
i test some simple svg 
it works ok but it too slow ，load 20 svgs nearly 10s onAndroid

**Comment by MrJoy** [comment]
This is true.  Unfortunately, I'm not in a position to make the major architectural changes needed to make it faster.
